packages/utils/src/__tests__/WorkBookParser.test.js

```diff
@@ -55,9 +58,10 @@ describe('WorkBookParser', () => {
       });
     });
 
-    it('should map the input to itself if no mapper is provided', () => {
+    it('should map the input to itself if no mapper is provided', async () => {
       const parser = new WorkBookParser();
-      return expect(parser.parse(WORK_BOOK)).to.eventually.deep.equal(SHEETS);
+      const result = await parser.parse(WORK_BOOK);
+      return expect(result).toEqual(SHEETS);
```

Comment:
Although the above syntax is totally fine, let's try to convert it to:

```suggestion
      return expect(parser.parse(WORK_BOOK)).resolves.toStictEqual(SHEETS);
```

since it is more similar to the non asynx `expect` syntax, retaining an input => output flow

Similarly for the other test cases in this file where we `await` results and then `expect` on them